### ISSUE TITLE
Support for nested Placeholder Tags

### ISSUE THREAD

**Issue by Euphonique** [PluginBugs/Issues-ItemsAdder#1942]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Describe the solution you'd like

Support for nested Placeholder-Tags would be great:

Something like %img_%player_world%% ?
Tried %img_{%player_world%}%, Tried %img_<%player_world%>% with and without %.

I've talked to the Developer of PlaceholderAPI and he said:
"You can ask LoneDev to add support for nested placeholders. All he had to do is use PlaceholderAPI.parseBracketPlaceholders(player, input) before dealing with the text itself."

### Is your feature request related to a problem?

no

### Describe alternatives you've considered

-

### Additional context
"It would be extremely difficult for PlaceholderAPI to provide support for nested placeholders, especially for the %-based one. Because the way it works right now is that PAPI goes through the entire text provided to find a %. If it does will it try to create a "valid" placeholder by finding text followed by a _, then text again and finally a %.

This means that a placeholder like %img_%player_world%% would result in PlaceholderAPI finding %img_% which it considers an invalid placeholder and therefore continues.
That's why some placeholder expansions offer support through PlaceholderAPI's alternate placeholder pattern, which is {identifier_args}. This requires the expansion to handle it tho.

So tl;dr: It's currently not possible to allow nested placeholders in PlaceholderAPI due to how it currently handles the placeholders to begin with.

You can ask LoneDev to add support for nested placeholders. All he had to do is use PlaceholderAPI.parseBracketPlaceholders(player, input) before dealing with the text itself."
[https://www.spigotmc.org/threads/placeholderapi.61918/page-148#post-4472893](https://www.spigotmc.org/threads/placeholderapi.61918/page-148#post-4472893)

**Comment by LoneDev6** [comment]
ItemsAdder is not the plugin which handles replacements, you have to ask your chat plugin developer to use the suggested method.
ItemsAdder is just registering the placeholders, other plugins need to use them.

Have a nice day
